### PR TITLE
WoE: implement Niffle Paw

### PR DESCRIPTION
**Pull request by strib**

### CODE REVIEW

File: server/game/cards/06-WoE/NifflePaw.js [hunk 1]
```diff
@@ -0,0 +1,27 @@
...
+                onReap: (event, context) => event.card === context.source.parent
+            },
+            gameAction: ability.actions.destroy((context) => ({ target: context.source.parent })),
+            then: {
+                alwaysTriggers: true,
+                target: {
+                    numCards: 1,
+                    cardType: ['creature'],
+                    cardCondition: (card) => card.location === 'play area',
+                    gameAction: ability.actions.attach((context) => ({
```
Comment (strib) [inline server/game/cards/06-WoE/NifflePaw.js:16]: I can't figure out why I'm not seeing a log message for this attachment action.  It's probably something simple.  Help appreciated!

Comment (romanogit) [reply server/game/cards/06-WoE/NifflePaw.js:16]: Because that's on a `then` statement. you have to provide a message explicitly.

Comment (strib) [reply server/game/cards/06-WoE/NifflePaw.js:17]: Thanks, fixed!

Comment (romanogit) [reply server/game/cards/06-WoE/NifflePaw.js:17]: message should follow the format "{0} uses {1} to move ... "... something like that....

**Review by romanogit — CHANGES_REQUESTED** [review]

File: server/game/cards/06-WoE/NifflePaw.js (hunk 1 above)
Comment (strib) [reply server/game/cards/06-WoE/NifflePaw.js:17]: So it's ok to have "uses Niffle Paw to move Niffle Paw to X"?  That seemed weird to me but happy to do it.

Comment (strib) [reply server/game/cards/06-WoE/NifflePaw.js:17]: Ok did it that way.

Comment (romanogit) [reply server/game/cards/06-WoE/NifflePaw.js:17]: Yes, that's how "effects" are worded. But in this case, I think it should say, for example:

"romano uses "creature" to move Niffle Paw to "another creature"

**Pull request merged**